applications/crossbar/priv/couchdb/schemas/functions.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "functions",
+    "description": "Functions are subscriptions to allowed events that, when the event occurs, the event data is sent to the function set in the function document.",
+    "properties": {
+        "enabled": {
+            "default": true,
+            "description": "Is the function enabled and running",
+            "type": "boolean"
+        },
+        "function_js": {
```

Comment:
maybe "definition"? Since it is a "function" schema, the "function"'s "definition" contains the code?

Reply:
this is still unpolished. I should have used object as type not string PROBABLY.